atd-vze/src/views/Locations/LocationMap.js

```diff
@@ -0,0 +1,132 @@
+import React, { Component } from "react";
+import MapGL, { NavigationControl, FullscreenControl } from "react-map-gl";
+import axios from "axios";
+import moment from "moment";
+
+import styled from "styled-components";
+import { Button } from "reactstrap";
+
+const TOKEN = process.env.REACT_APP_MAPBOX_TOKEN;
```

Comment:
we could add the `NEARMAP_TOKEN` here like we did for Mapbox

Reply:
Cool, I'll make that change before merge (minus the actual token).